src/main/resources/CucumberSuite.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version = "1.0" encoding = "UTF-8"?>
+<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >
+
+
+<suite name = "Exercises Hw 4">
+
+    <listeners>
+        <listener class-name="hw.four.listeners.AllureListener"></listener>
+    </listeners>
+
+    <test name = "TestViaCucumberAndTestNG">
+        <classes>
+            <class name = "hw.five.CucumberTestNGTest" />
+        </classes>
+    </test>
+
+</suite>
```

Comment:
Do you use this `CucumberSuite` anywhere? 'cause I can't find its usages
If so please check that there is no `CucumberTestNGTest` class in packages

Reply:
deleted